src/performance_profiles.jl

```diff
@@ -13,21 +13,22 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
 
   (np, ns) = size(T);       # Number of problems and number of solvers.
 
-  T[isinf(T)] = NaN;
-  T[T .< 0] = NaN;
+  T[isinf.(T)] = Inf;
```

Comment:
You're right it should be `isnan` instead of is inf. But I don't see why `maximum` should be kept. At least on my tests with `isnan` the `maximum` function still produce bad performance profiles